src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/DisposeAnalysis/DisposeAnalysis.cs

```diff
@@ -82,7 +83,7 @@ private DisposeAnalysis(DisposeAnalysisDomain analysisDomain, DisposeDataFlowOpe
             bool exceptionPathsAnalysis,
             PointsToAnalysisKind pointsToAnalysisKind,
             bool performCopyAnalysis,
-            SymbolNamesWithValueOption<Unit> excludedSymbols,
+            Func<ISymbol, bool> isConfiguredToSkipAnalysis,
```

Comment:
Let me know if passing a `Func` is a performance issue. Also, let me know if you prefer a delegate to have something maybe more explicit.

Reply:
Should be fine.